File: CA2API/CA2API/Match.cs
using System;

namespace HelloWorldWebAPI.Models
{
    // a simple greeting
    public class Match
    {
        public int ID { get; set; }
        public Team Home { get; set; }
        public Team Away { get; set; }
        //public int HomeScore { get; set; }
        //public int AwayScore { get; set; }
    }
}

File: ca2app/Match.java
package org.gc.helloworldvolleyclient;

public class Match {
    public int ID;
    public Team Home = new Team();
    public Team Away = new Team();
    //public int HomeScore;
    //public int AwayScore;

    public String toString()
    {
        return ID + " " + Home + " " + Away;
    }
}
